src/components/IssueGraph.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { useQuery } from '@tanstack/react-query';
+import { useParams } from '@tanstack/react-router';
+import { Chart as ChartJS, ArcElement, Tooltip, Legend } from 'chart.js/auto';
+import { Bar } from 'react-chartjs-2';
+import { getClosedIssueCount, getOpenIssueCount } from '../services/StatsAPI';
+import { RouteParams } from '../models';
+
+const IssueGraph = () => {
+  ChartJS.register(ArcElement, Tooltip, Legend);
+
+  const {username, repo} : RouteParams = useParams({ strict: false });
+
+  const OpenIssueQuery = useQuery({
+    queryKey: [`getOpenIssueCount`, username, repo],
+    queryFn: () => getOpenIssueCount(username, repo),
+  });
+
+  const ClosedIssueQuery = useQuery({
+    queryKey: [`getCompletedIssueCount`, username, repo],
+    queryFn: () => getClosedIssueCount(username, repo),
+  });
+
+  const data = {
+    labels: ['Open', 'Completed'],
+    datasets: [
+      {
+        label: 'Issues',
+        data: [OpenIssueQuery.data, ClosedIssueQuery.data],
+        backgroundColor: ['red', 'green'],
+        borderColor: 'transparent',
+        borderWidth: 1,
+      },
+    ],
+  };
+
+  const options = {
+    scales: {
+      y: {
+        beginAtZero: true,
+      },
+    },
+  };
+
+  if (OpenIssueQuery.isLoading || ClosedIssueQuery.isLoading) {
```

Comment:
Just FYI from what i've been told... Apparently for UseQuery we should use both isLoading and isPending

Reply:
Okay, thank you. I will add it in my next branch where I'm doing skeleton loading